app/helpers/medications_helper.rb

```diff
@@ -11,18 +12,79 @@ def edit_medication_props
     edit_form_props(medication_form_inputs, medication_path(@medication))
   end
 
+  def medications_props(medications)
+    medications.map { |medication| present_medication(medication) }
+  end
+
+  # rubocop:disable MethodLength
+  def present_medication(medication)
+    {
+      name: medication.name,
+      link: medication_path(medication),
+      actions: {
+        edit: {
```

Comment:
I think we can have the action in private methods:-
1. Reduce length of the method
2. In case we need another action, it will be pretty easy to add it as `xx_action` and have a method that returns the action

Reply:
I think this is a great piece of feedback we should address!